docs/development/writingzeppelininterpreter.md

```diff
@@ -14,7 +14,7 @@ Interpreters in the same InterpreterGroup can reference each other. For example,
 
 <img class="img-responsive" style="width:50%; border: 1px solid #ecf0f1;" height="auto" src="../../assets/themes/zeppelin/img/interpreter.png" />
 
-Interpreter can be launched either using separate classloader or separate JVM process. Sometimes separate classloader causes problem especially when your interpreter uses reflections or trying to grab standard out/err. In this case, separate JVM process is the option you can select. (by checking 'fork' in Interpreter menu, which is default value) When Interpreter is running in separate JVM process, it's communicating with Zeppelin via thrift.
+All Interpreters in the same interpreter group are launched in a single, separate JVM process. The Interpreter communicates with Zeppelin engine via thrift.
```

Comment:
OK, so we are done. Can you marge this please?
Eran
 (My First contribution :-))

On Thu, Jul 16, 2015 at 4:17 PM Lee moon soo <notifications@github.com>
wrote:

> In docs/development/writingzeppelininterpreter.md
> <https://github.com/apache/incubator-zeppelin/pull/155#discussion_r34785207>
> :
>
> > @@ -14,7 +14,7 @@ Interpreters in the same InterpreterGroup can reference each other. For example,
> >
> >  <img class="img-responsive" style="width:50%; border: 1px solid #ecf0f1;" height="auto" src="../../assets/themes/
>
> zeppelin/img/interpreter.png" />
> >
> > -Interpreter can be launched either using separate classloader or separate JVM process. Sometimes separate classloader causes problem especially when your interpreter uses reflections or trying to grab standard out/err. In this case, separate JVM process is the option you can select. (by checking 'fork' in Interpreter menu, which is default value) When Interpreter is running in separate JVM process, it's communicating with Zeppelin via thrift.
> > +All Interpreters in the same interpreter group are launched in a single, separate JVM process. The Interpreter communicates with Zeppelin engine via thrift.
>
> Yes, true [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/155/files#r34785207>.
>